_inc/client/main.jsx

```diff
@@ -267,6 +268,19 @@ class Main extends React.Component {
 	}
 
 	render() {
+		if ( this.props.route.path === '/plans-prompt' ) {
+			return (
+				<div>
+					<div className="jp-lower">
+						<AdminNotices />
+						<JetpackNotices />
+						<PlansPrompt siteAdminUrl={ this.props.siteAdminUrl } />
+					</div>
+					<Tracker analytics={ analytics } />
```

Comment:
We're now displaying the footer here. Is that on purpose?

Reply:
The mock-up for the plans prompt doesn't show the footer, but we could potentially add it, I think. Let's ask @jeffgolenski for advise.